virtool/tasks/runner.py

```diff
@@ -19,24 +19,42 @@ def __init__(
         self._data = data
         self._tasks_client = tasks_client
         self.app = app
+        self.current_task = None
```

Comment:
Add type annotation.
```suggestion
        self._current_task = None
```